ironfish-cli/src/commands/wallet/assets.ts

```diff
@@ -100,6 +100,14 @@ export class AssetsCommand extends IronfishCommand {
                 ? BufferUtils.toHuman(Buffer.from(row.creator, 'hex'))
                 : row.creator,
           },
+          owner: {
+            header: 'Owner',
+            minWidth: PUBLIC_ADDRESS_LENGTH + 1,
```

Comment:
Not in this PR, but perhaps we should think about shortening the `minWidth` of both creator and owner